api/v1alpha1/condition_types.go

```diff
@@ -23,10 +23,11 @@ type Condition struct {
 type ConditionType string
 
 const (
-	ConditionActive   ConditionType = "Active"
-	ConditionCompiled ConditionType = "Policy"
-	ConditionSecret   ConditionType = "Key"
-	ConditionListener ConditionType = "Listener"
+	ConditionActive      ConditionType = "Active"
+	ConditionCompiled    ConditionType = "Policy"
+	ConditionSecret      ConditionType = "Key"
+	ConditionInitialized ConditionType = "Stream"
```

Comment:
Open to suggestions on the naming here.

Reply:
I think it is less confusing to just use the value in the name `ConditionStream ConditionType = "Stream"`

I also think we could use the same Condition for the Attester and Enforcer resources. So just replace ConditionListener with ConditionStream in the Enforcer Controller

Reply:
@ChrisSchreiber makes sense, I updated the enforcer controller and its CRDs